bin/compare.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Added this coz there were some errors but seeing them in GHA was hard, and no way to easily recreate them locally